### ISSUE TITLE
[Bug] "Trigger Autosave" seems to stop working when a test passes 2000 entities

### ISSUE THREAD

**Issue by cartman1337** [TeamSpen210/BEE2-items#892]
I'm working on a test, which turns out to be huge. In fact, it just passed 2000 entities in the compiler, but still below the hard limit of 2048. Up until that point, the "Trigger Autosave" points I had put in was working, but after reaching that limit, I started getting compile errors referencing a missing file connection for "trigger_autosave.vmf".

For some reason, the "Trigger Once" points I had were still working, which is part of the same item type.

Fortunately, the separate "Save Point" item does the same job, and compiling with those worked. On that note I can contribute that one "Save Point" costs 1 entity in the compiler. Currently that item is listed with "?? entities" in Bee2.exe.

**Comment by BenVlodgi** [comment]
Got a puzzle file to easily test this with?

**Comment by cartman1337** [comment]
Just close the issue. I'm unable to reproduce it again. At the time I got it, no matter what I did it refused to compile with "Trigger Autosave" in the map, but worked with "Save Point", but "Trigger Autosave" was still working in other maps. Now, if I replace the "Save Points" with "Trigger Autosave" again in the map that failed, it now compiles. Between those times the computer had been restarted for Windows Updates, with new full exports of Bee2 packages after that, so the fault was probably limited to that one instance.
